app/src/main/java/com/infomaniak/mail/data/cache/mailboxContent/ThreadController.kt

```diff
@@ -71,6 +71,11 @@ class ThreadController @Inject constructor(
         return getThreadQuery(uid, mailboxContentRealm()).asFlow()
     }
 
+    fun getMessageCountInThreadForFolder(threadUid: String, folderId : String) : Int{
```

Comment:
This function is getting data about Messages, so move it to the MessageController.